docs/contributor-guide/component/external_tools.md

```diff
@@ -0,0 +1,142 @@
+# Components Using External Tools
+
+It often makes sense to write a component which makes use of third-party tools in one way or 
+another. If these tools are distributed in Python modules, the module can simply be added to the 
+requirements in the OFRAK package's `setup.py` file. But for dependencies on tools which cannot be
+simply installed with `pip`, OFRAK has another mechanism for the making the dependency clear to users
+and easy to install.
+
+### ComponentExternalTool
+
+`ofrak.model.component_model.ComponentExternalTool` is a class that encapsulates the information 
```

Comment:
Should this use the MkDocs cross reference syntax? I don't know if that syntax will work outside of docstrings...

Reply:
Yes it should have, I was using docstring format indeed